nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -486,7 +748,10 @@ def generate_aten_to_python(func: Callable, node: NodePyGroup, speedup: ModelSpe
     c_node = node.key_node
 
     schema = c_node.schema()
-    positional_num, keyword_list, special_treat = parse_aten_schema(schema)
+    if torch.__version__ < '1.9.0':
```

Comment:
![image](https://user-images.githubusercontent.com/33053116/196076823-8e2f342e-562f-4049-bbee-d49666af1d56.png)

I'm worried with this method of judging version.

Reply:
the type of `torch.__version__` is not a `str`. it's a version type that can correctly compare with str.